.github/workflows/integration_tests.yaml

```diff
@@ -36,11 +36,11 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -e projects/fal
-          pip install pytest pytest-asyncio pillow
+          pip install pytest pytest-asyncio pytest-xdist pillow
```

Comment:
I will add it as a `fal[dev]` along with pytest section in pyprojects a little bit later (want to get rid of poetry too first).